### PR TITLE
[AP-2] - changed group_id and version in pom.xml

### PR DESCRIPTION
**Pull request by vlad-lisnenko**
I have changed group_id and version in pom.xml

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,9 +4,9 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.example</groupId>
+    <groupId>com.autos</groupId>
     <artifactId>auto-project</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0</version>
```
Comment (scratchw0w) [inline pom.xml:9]: can you please explain, why do we need to remove snapshot word?

File: pom.xml [hunk 2]
```diff
@@ -4,9 +4,9 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.example</groupId>
+    <groupId>com.autos</groupId>
```
Comment (scratchw0w) [inline pom.xml:7]: and why we need autos here?

File: pom.xml (hunk 1 above)
Comment (vlad-lisnenko) [reply pom.xml:9]: because this version is not  snapshot

File: pom.xml (hunk 2 above)
Comment (vlad-lisnenko) [reply pom.xml:7]: because of package

File: pom.xml (hunk 1 above)
Comment (scratchw0w) [reply pom.xml:9]: thanks, i've understood

**Review by scratchw0w — APPROVED** [review]
seems good to me

**Pull request merged**